src/microbe_stage/systems/PilusDamageSystem.cs

```diff
@@ -11,7 +11,6 @@
     [With(typeof(OrganelleContainer))]
     [With(typeof(CollisionManagement))]
     [With(typeof(MicrobePhysicsExtraData))]
-    [With(typeof(Species))]
```

Comment:
Ah, I've made a serious typo :facepalm: 
This line should be `    [With(typeof(SpeciesMember))]` as this system reads the species member component of the entity.